apps/console/src/app/pages/organizations/OrgPage.tsx

```diff
@@ -83,7 +84,7 @@ export const OrgPage = () => {
       ),
     [tabsItems, isOrgAdmin]
   );
-
+  usePageTitle(`${organization?.name || "Organization"}`);
```

Comment:
```suggestion
  usePageTitle(organization?.name);
```
With the fallback we don't need the `||` here.